### ISSUE TITLE
In case of a missing property, which http status should be returned?

### ISSUE THREAD

**Issue by tomkuijsten** [ocpi/ocpi#280]
From the 2.2-d2 status_codes document:

_If a request is syntactically valid JSON and addresses an existing resource, **a HTTP error MUST NOT be returned**. Those requests are supposed to have reached the OCPI layer.

In case of a GET request, when the resource does NOT exist, the server SHOULD return a HTTP 404 - Not Found.

When the server receives **a valid OCPI object** it SHOULD respond with:

HTTP 200 - Ok when the object already existed and has successfully been updated.

HTTP 201 - Created when the object has been newly created in the server system.

Requests that reach the OCPI layer **SHOULD return an OCPI response message** with a status_code field as defined below._

So, which http status should I send when it sends valid json, but invalid json data (eg property missing)? An http error status would conflict with the first bold part. The 200 or 201 with a response object with an status_code (2xxx range) would be very weird from REST perspective.

**Comment by RobertdeLeeuw-IHomer** [comment]
Yes, it is not really "standard" REST. But all these "client-owned-object" are not REST either. Try to see OCPI as a protocol using REST as the transport layer. In the OSI model it is more common to handle error at the correct level.
